### ISSUE TITLE
Is pkce also supported?

### ISSUE THREAD

**Issue by BasBerk** [fedora-infra/flask-oidc#44]
I was wondering is pkce flow also supported?

**Comment by willthetanman** [comment]
I don't think this package supports PKCE, however for PKCE auth flow we use this https://github.com/CamiloTerevinto/TerevintoSoftware.PythonPkceClient.
